node/network/bridge/src/network.rs

```diff
@@ -175,6 +183,14 @@ impl Network for Arc<NetworkService<Block, Hash>> {
 		NetworkService::event_stream(self, "polkadot-network-bridge").boxed()
 	}
 
+	async fn add_to_peers_set(&mut self, protocol: Cow<'static, str>, multiaddresses: HashSet<Multiaddr>) -> Result<(), String> {
+		sc_network::NetworkService::add_to_peers_set(&**self, protocol, multiaddresses)
```

Comment:
hmm, I'm confused, does it take `out_peers` into account? The reason we used `reserved` is that we want to connect to all peers, but limit outgoing messages on a higher level

Reply:
We always try connect to all the nodes that have been added, `out_peers` isn't taken into account.

Reply:
what's the difference with `add_peers_to_reserved_set` then?

> out_peers isn't taken into account

https://github.com/paritytech/substrate/blob/cdc1bb10cf69115ad56fbf5b0ce2fa4ddf8dc02e/client/peerset/src/lib.rs#L546

but here we do, don't we?

Reply:
Ah dammit, that change wasn't intended